frontend/packages/kubevirt-plugin/integration-tests/tests/utils/constants/common.ts

```diff
@@ -44,13 +44,13 @@ export const VIRTUALIZATION_TITLE = 'Virtualization';
 export const IMPORT_WIZARD_CONN_TO_NEW_INSTANCE = 'Connect to New Instance';
 export const NOT_RECOMMENDED_BUS_TYPE_WARN = 'Not recommended bus type';
 // Some times we need to use existing VMWare instance, which name always starts from 'administrator'
-export const IMPORT_WIZARD_CONN_NAME_PREFIX = 'administrator';
+export const IMPORT_WIZARD_CONN_NAME_PREFIX = 'admin';
 export const RHV_PROVIDER = 'Red Hat Virtualization (RHV)';
 export const VMWARE_PROVIDER = 'VMware';
 
 // Web-UI Exceptions
 export const RHV_WIZARD_CREATE_SUCCESS = 'Started import of virtual machine';
-export const WIZARD_CREATE_SUCCESS = 'Successfully created virtual machine';
+export const WIZARD_CREATE_SUCCESS = 'Started import of virtual machine';
```

Comment:
this will break all the other flows

Reply:
This will not, it didn't work before and replacing this string made the fix. Locally everything works perfectly!

Reply:
It will fail the other wizard tests. And as I said in the main comment, it might be working for your environment, but it fails on mine.

Reply:
updated, added `VMWARE_WIZARD_CREATE_SUCCESS' for my own tests and reverted value of `WIZARD_CREATE_SUCCESS` back